db/migrations/20250122190038_add_warning_status_and_sub_status_to_work_items.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex, Promise) {
+  return knex.schema.raw(`
+    ALTER TABLE "work_items"
+    DROP CONSTRAINT "work_items_status_check",
+    ADD CONSTRAINT "work_items_status_check"
+    CHECK (status IN ('ready', 'queued', 'running', 'successful', 'failed', 'canceled', 'warning')),
+    ADD COLUMN "sub_status" VARCHAR(255);
+
+    CREATE INDEX work_items_sub_status ON work_items (sub_status);
```

Comment:
nit: add `_index` to index name for consistency. i.e. `work_items_sub_status_index`.

Reply:
Done